packages/bun-usockets/src/context.c

```diff
@@ -63,8 +62,9 @@ void us_socket_context_close(int ssl, struct us_socket_context_t *context) {
     /* Begin by closing all listen sockets */
     struct us_listen_socket_t *ls = context->head_listen_sockets;
     while (ls) {
-        struct us_listen_socket_t *nextLS = (struct us_listen_socket_t *) ls->s.next;
+        struct us_listen_socket_t *nextLS = (struct us_listen_socket_t *) ls->s.next;\
```

Comment:
C is one of the only languages where you can have an accidental `\` and nothing breaks

Reply:
```suggestion
        struct us_listen_socket_t *nextLS = (struct us_listen_socket_t *) ls->s.next;
```

Reply:
surprised that it compiled